runtime/src/main/java/org/corfudb/runtime/view/stream/ReplexStreamView.java

```diff
@@ -159,7 +159,7 @@ public ILogData previous() {
         doPendingSeek(getCurrentContext(), Address.MAX);
 
         // If never read or only one entry, return null
-        if (getCurrentContext().streamPointer == Address.NEVER_READ ||
+        if (getCurrentContext().streamPointer <= Address.maxNonAddress() ||
```

Comment:
I think we should pull these comparisons into the Address class. The operators need not be exposed.